### PR TITLE
b/188685156: Fix presubmit failure by pex build

### PR DESCRIPTION
**Pull request by TAOXUY**
Enforce the `markupsafe` version in pex_binary.reqs. The latest markupsafe will cause `pex.resolver.Untranslateable`.

### CODE REVIEW

**Comment by TAOXUY** [comment]
Hi @qiwzhang, the `//src/nginx/t:grpc_interop_upstream_ssl` is failed and this is the error log
```
E0520 03:35:52.609727291    9659 ssl_transport_security.cc:1238] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0520 03:35:52.610348866    9665 interop_client.cc:138]      Error status code: 14 (expected: 0), message: Socket closed, debug string: {"created":"@1621481752.610317710","description":"Error received from peer ipv4:127.0.0.1:9380","file":"external/com_github_grpc_grpc/src/core/lib/surface/call.cc","file_line":1046,"grpc_message":"Socket closed","grpc_status":14}
```

You can reproduce by using the doc I just added. Do you have any idea on this?

File: doc/debug_presubmit.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ESP runs the presubmit unit/integration/e2e tests on Jenkins. The local tests pass
+may not necessarily make ESP presubmit pass as different environment. The most common reasons is
+some dependency incompatibility.
+
+## How to locally reproduce the presubmit failure
+
+#### Getting the presubmit image
+The address of docker image used to run presubmit is is located in [Jenkinsfile](https://github.com/cloudendpoints/esp/blob/master/Jenkinsfile) under var `SLAVE_IMAGE`, like 
+```
+SLAVE_IMAGE = 'gcr.io/endpoints-jenkins/debian-9:0.13'
```
Comment (nareddyt) [inline doc/debug_presubmit.md:12]: Do you know how this image is created? It is 1 year old, and dependencies inside it that are outdated?

**Review by qiwzhang — APPROVED** [review]
Cool.  Thanks.

File: doc/debug_presubmit.md (hunk 1 above)
Comment (qiwzhang) [reply doc/debug_presubmit.md:12]: It is in this [Makefile](https://github.com/cloudendpoints/esp/blob/master/jenkins/slaves/Makefile)

**Comment by qiwzhang** [comment]
we can fix //src/nginx/t:grpc_interop_upstream_ssl  test failure in a separate pr.

**Pull request merged**